src/Data/Semigroup/Cancellative.hs

```diff
@@ -53,11 +54,6 @@ import qualified Data.Set as Set
 import qualified Data.Vector as Vector
 import Numeric.Natural (Natural)
 
--- | Class of all Abelian (/i.e./, commutative) semigroups that satisfy the commutativity property:
--- 
--- > a <> b == b <> a
-class Semigroup m => Commutative m
-
 -- | Class of Abelian semigroups with a partial inverse for the Semigroup '<>' operation. The inverse operation '</>' must
```

Comment:
With the class definition leaving this file, this haddock no longer defines "Abelian" in a convenient place for any readers who are not familiar with Group Theory.

Reply:
@endgame I am still reexporting `Commutative`. I think we can also move the haddocks to the reexport? I didn't think of that before.

Reply:
I think if you re-export and you put it first in the export list, it will be a well-ordered read.

Reply:
I scrolled up, and `Commutative` is in fact first!

Reply:
That's easy, then. If we're not changing the set of things exported by the module, I think a patch revision bump (the fourth component in the PVP version number) is appropriate. We are not changing the interface to this package.

Reply:
The third component please. We are adding a whole new dependency after all.

Reply:
I am not sure to what extent changing dependencies effects the version under the PVP at all, but regardless, you are of course free to bump or not bump the version of your own library as you please.

Reply:
Actually just leave the version bump to me, I'm planning some other small changes for the next release anyway.

Reply:
Yeah leaving it to you was always my intent :).